.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,7 +10,7 @@ The project is undergoing daily changes. Pull Requests will be reviewed and resp
 
 ### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?
 
-(Write your answer here.)
+(We must have a signed CLA that includes your GitHub handle prior to accepting and landing your work. Write your answer here.)
```

Comment:
link the CLA doc here again? I think many people don't really read the contribution guidelines